perun-cli/getAttribute

```diff
@@ -12,57 +12,110 @@ sub help {
 	Prints selected attribute for an entity.
 	---------------------------------------------------
 	Available options:
-	--userId       | -u  user identifier
-	--facilityId   | -f  facility identifier
-	--memberId     | -m  member identifier
-	--resourceId   | -r  reource identifier
-	--hostId       | -h  host identifier
-	--groupId      | -g  group identifier
-	--voId         | -v  vo identifier
-	--attributeId  | -a  attribute identifier
-	--orderById    | -i  order by numeric Id
-	--orderByName  | -n  order by Name
-	--batch        | -b  batch
-	--help         | -h  prints this help
+	--userId          | -u  user identifier
+	--facilityId      | -f  facility identifier
+	--memberId        | -m  member identifier
+	--resourceId      | -r  reource identifier
+	--hostId          | -t  host identifier
+	--groupId         | -g  group identifier
+	--voId            | -v  vo identifier
+	--userExtSourceId | -x  user ext source identifier
+	--entityless      | -e  subject of entityless attribute
+	--attributeId     | -a  attribute identifier
+	--attributeName   | -A attribute name (including namespace)
+	--orderById       | -i  order by numeric Id
+	--orderByName     | -n  order by Name
+	--batch           | -b  batch
+	--help            | -h  prints this help
 	};
 }
 
-my ($userId, $facilityId, $memberId, $resourceId, $groupId, $voId, $hostId, $attributeId, $batch, $sortingFunction);
+my ($userId, $facilityId, $memberId, $resourceId, $groupId, $voId, $hostId, $entitylessSubj, $uesId,$attributeId, $attributeName, $attributeDefinition, $batch, $sortingFunction, @attributes);
```

Comment:
I would prefer defining ``@attributes`` variable later, when needed. This is a long line already and usually defines only variable for reading user input. ``$attributeDefinition`` is probably the same case.